modules/middleware/repo.go

```diff
@@ -16,6 +16,14 @@ import (
 	"github.com/gogits/gogs/modules/git"
 	"github.com/gogits/gogs/modules/log"
 	"github.com/gogits/gogs/modules/setting"
+
+	"github.com/mcuadros/go-version"
```

Comment:
I would recommend to use https://github.com/hashicorp/go-version which has methods like `LessThan` `GreaterThan`, Gogs may use this one in the future, so it's better to start using it now.